ui/src/app/components/runtime-configuration-panel.tsx

```diff
@@ -228,31 +189,13 @@ const PanelMain = fp.flow(
       }
     }, [analysisConfig.computeType]);
 
-    const runtimeExists = (status && isVisible(status)) || !!pendingRuntime;
-    const disableControls = runtimeExists && !isActionable(status);
```

Comment:
moved inside customize-panel

Reply:
also true for dataprocExists, unattachedPdExists, and unattachedDiskNeedsRecreate

Reply:
DO you feel we can add comments on some scenarios where isVisible will be true?